python/ml4ir/applications/ranking/tests/test_serving.py

```diff
@@ -19,6 +19,7 @@ def test_model_serving(self):
 
         # Test model training on TFRecord SequenceExample data
         data_dir = os.path.join(self.root_data_dir, "tfrecord")
+        self.feature_config_fname = "feature_config_integration_test.yaml"
```

Comment:
@jakemannix @ducouloa Using the same feature config for both python-jvm integration tests and internal python(training)-python(serving) integration test.